### PR TITLE
Mepts 830 tx curr differences in ccs db

### PR DESCRIPTION
**Pull request by cfaife**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXCurrQueries.java [hunk 1]
```diff
@@ -471,120 +471,81 @@ public static String getPatientWithoutScheduledDrugPickupDateMasterCardAmdArtPic
...
+            + "WHERE  pat.voided=0 "
+            + "    AND    pat.patient_id NOT IN( "
+            + "        SELECT fn.patient_id "
+            + "        FROM  ( "
+            + "            SELECT     qa.patient_id "
+            + "            FROM ( "
+            + "                SELECT pat.patient_id,Max(e.encounter_datetime) AS encounter_datetime "
+            + "                FROM  patient pat "
+            + "                    INNER JOIN encounter e ON  pat.patient_id=e.patient_id "
+            + "                WHERE      e.encounter_datetime<= :onOrBefore "
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXCurrQueries.java]: Put some space after the e.encounter_datetime before the <= only helps for readability and also a line above it

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXCurrQueries.java [hunk 2]
```diff
@@ -471,120 +471,81 @@ public static String getPatientWithoutScheduledDrugPickupDateMasterCardAmdArtPic
...
+            + "                    AND        pat.voided=0 "
+            + "                    AND        e.voided=0 "
+            + "                    AND        e.location_id= :location "
+            + "                    AND        e.encounter_type IN(${6},${9}) "
+            + "                GROUP BY   pat.patient_id "
+            + "                ) qa "
+            + "                INNER JOIN encounter e1 ON qa.patient_id=e1.patient_id "
+            + "                INNER JOIN obs o1 ON e1.encounter_id=o1.encounter_id "
+            + "            WHERE      qa.encounter_datetime=e1.encounter_datetime "
+            + "                AND        e1.encounter_datetime<= :onOrBefore "
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXCurrQueries.java]: Put some space after the e.encounter_datetime before the <= only helps for readability

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXCurrQueries.java [hunk 3]
```diff
@@ -471,120 +471,81 @@ public static String getPatientWithoutScheduledDrugPickupDateMasterCardAmdArtPic
...
+            + "                AND        o1.voided=0 "
+            + "                AND        o1.concept_id IN(${1410}) "
+            + "                AND        o1.location_id = :location "
+            + "            UNION "
+            + "            SELECT     qc.patient_id "
+            + "            FROM ( "
+            + "                SELECT     pat.patient_id, max(e.encounter_datetime) AS encounter_datetime "
+            + "                FROM       patient pat "
+            + "                    INNER JOIN encounter e ON         pat.patient_id=e.patient_id "
+            + "                WHERE      e.encounter_datetime<= :onOrBefore "
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXCurrQueries.java]: Put some space after the e.encounter_datetime before the <= only helps for readability

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXCurrQueries.java [hunk 4]
```diff
@@ -471,120 +471,81 @@ public static String getPatientWithoutScheduledDrugPickupDateMasterCardAmdArtPic
...
+            + "                    AND        pat.voided=0 "
+            + "                    AND        e.voided=0 "
+            + "                    AND        e.location_id= :location "
+            + "                    AND        e.encounter_type IN(${18}) "
+            + "                GROUP BY   pat.patient_id "
+            + "              )qc "
+            + "                INNER JOIN encounter e1 ON         qc.patient_id=e1.patient_id "
+            + "                INNER JOIN obs o1 ON         e1.encounter_id=o1.encounter_id "
+            + "            WHERE      qc.encounter_datetime=e1.encounter_datetime "
+            + "                AND        e1.encounter_datetime<= :onOrBefore "
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXCurrQueries.java]: Put some space after the e.encounter_datetime before the <= only helps for readability

**Review by ningosi — COMMENTED** [review]
The codebase looks good, however, please provide an extra space after the variable, the the arithmetic operation, just for readability, though that can NOT stop the show, it is a recommended practice

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXCurrQueries.java (hunk 4 above)
Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXCurrQueries.java]: done

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXCurrQueries.java (hunk 3 above)
Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXCurrQueries.java]: done

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXCurrQueries.java (hunk 2 above)
Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXCurrQueries.java]: done

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXCurrQueries.java (hunk 1 above)
Comment (cfaife) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TXCurrQueries.java]: done

**Review by ningosi — APPROVED** [review]

**Pull request merged**